sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -291,6 +291,49 @@ certificate will be added with the alias of `mycert`.
 2. Certificates coming from Azure Key Vault take precedence over 
 side-loaded certificates.
 
+### Enable mutual TLS on the server side
+
+Only some minor changes need to be done to the server side SSL example 
+mentioned above.
+
+The following additional application.properties need to be added:
+
+```
+server.ssl.client-auth=need
+server.ssl.trust-store-type=AzureKeyVault
+```
+
+### Enable mutual TLS on the client side
+
+Only some minor changes need to be done to the client side SSL example 
+mentioned above.
+
+1. The SSL context needs to take a ClientPrivateKeyStrategy
+
+An example is shown below:
+
+```java
+SSLContext sslContext = SSLContexts.custom()
+        .loadKeyMaterial(ks, "".toCharArray(), new ClientPrivateKeyStrategy())
+        .loadTrustMaterial(ks, new TrustSelfSignedStrategy())
+        .build();
+```
+
+2. A ClientPrivateKeyStrategy needs to be defined.
+
+An example is show below:
+
+```java
+private class ClientPrivateKeyStrategy implements PrivateKeyStrategy {
+
+    @Override
+    public String chooseAlias(Map<String, PrivateKeyDetails> map, Socket socket) {
+        return "self-signed";
+    }
+}
+```
```

Comment:
Could you please add these java code fragment into a java class? Just like `KeyVaultJcaManagedIdentitySample`.